projects/cashmere-examples/src/lib/cashmere-example.module.ts

```diff
@@ -0,0 +1,358 @@
+
```

Comment:
This shouldn't be checked in, it is part of the autogenerated files. 

https://github.com/HealthCatalyst/Fabric.Cashmere/pull/681#pullrequestreview-197126718

Reply:
I asked about adding it to .gitignore and @joeskeen said no?